torch/csrc/utils/python_numbers.h

```diff
@@ -89,6 +89,21 @@ inline int64_t THPUtils_unpackIndex(PyObject* obj) {
   return THPUtils_unpackLong(obj);
 }
 
+inline bool THPUtils_checkBool(PyObject* obj) {
+  auto result = PyBool_Check(obj);
+  return result;
```

Comment:
nit: you can just return PyBool_Check; btw you don't actually need this function, the functions in here are to handle python2-vs-python3, but I don't really object if you want to keep it.